app/src/androidTest/java/com/sdp/movemeet/Login/LoginActivityTest.java

```diff
@@ -77,41 +77,24 @@
     }
 
     @Test public void Register() {
-
         onView(withId(R.id.text_view_create_account)).perform(click());
     }
 
+    @Test
+    @LargeTest
+    public void Login_TrueNonEmpty(){
+        onView(withId(R.id.edit_text_email))
+                .perform(typeText(Email), closeSoftKeyboard());
+        onView(withId(R.id.edit_text_password))
+                .perform(typeText(Password), closeSoftKeyboard());
+        onView(withId(R.id.button_login)).perform(click());
+    }
+
     @After
     public void signOut() {
         FirebaseAuth fAuth = FirebaseAuth.getInstance();
         if (fAuth.getCurrentUser() != null) {
             fAuth.signOut();
         }
     }
-
```

Comment:
I think it's just to remove the blank spaces 😂 and change the Login_TrueNonEmpty test that was commented out because it wasn't working. But now everything is fine!